flink/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "display_name": "flink",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "flink",
+  "metric_prefix": "flink.",
+  "metric_to_check": "",
```

Comment:
Can we add a metric to check here? This lets the tile get marked as `Installed` if the metrics are being submitted.